deltachat-ios/Controller/ChatViewController.swift

```diff
@@ -195,6 +195,9 @@ class ChatViewController: MessagesViewController {
             }
         }
 
+        // maybe a message was added while the user was in the contact / chat profile
+        refreshMessages()
```

Comment:
hm, this loads the message list twice when opening the chat normally, or? one time in here and a second time in viewWillAppear()-> loadFirstMessages().

also, the refreshMessags() does not change the message count.

so, i suggest to use loadFirstMessages()  here and strike the other call for loadFirstMessages()

Reply:
did  a commit with the changes while playing/testing, https://github.com/deltachat/deltachat-ios/pull/572 , @cyBerta feel free to merge it to this pr or to close it and do sth else :)